tests/plugins/UnitTests/BuildServerIntegration/GitlabIntegration.Tests/GitlabRemoteParserTests.cs

```diff
@@ -43,5 +43,20 @@ public void Should_fail_in_parsing_invalid_url()
 
             gitlabRemoteParser.IsValidRemoteUrl(url).Should().BeFalse();
         }
+
+        [Test]
+        public void Should_Correct_in_parsing_repositoryNamespace()
+        {
+            GitlabRemoteParser gitlabRemoteParser = new();
+            string url = "https://dev.bad.com/groupname/groupname/repo.git";
+            gitlabRemoteParser.TryExtractGitlabDataFromRemoteUrl(url, out string? host, out string? owner, out string? repository).Should().BeTrue();
+            string? repositoryNamespace = owner;
+            repositoryNamespace.Should().NotBeNull();
+            repositoryNamespace.Should().BeEquivalentTo("groupname/groupname");
+            repositoryNamespace.Should().NotBeEquivalentTo("groupname");
+            repository.Should().BeEquivalentTo("repo");
+            host.Should().BeEquivalentTo("dev.bad.com");
```

Comment:
```
            repositoryNamespace.Should().NotBeNull();
            repositoryNamespace.Should().NotBeEquivalentTo("groupname");
``` are useless because compared to "groupname/groupname", right!?!

```suggestion
            host.Should().BeEquivalentTo("dev.bad.com");
            repositoryNamespace.Should().BeEquivalentTo("groupname/groupname");
            repository.Should().BeEquivalentTo("repo");
```